### ISSUE TITLE
Define on-topic vs off-topic explicitly, perhaps with examples

### ISSUE THREAD

**Issue by typesanitizer** [mgaudet/CompilerJobs#54]
I know the repository is titled "Compiler Jobs" and the README says "compiler, language and runtime teams", but it's a little unclear to me where exactly the line is drawn. Here are some examples:
- Are debugger engineer (e.g. Replay) openings on-topic? I see there is one mention of LLDB in the doc.
- Are [code intelligence](https://docs.sourcegraph.com/code_intelligence) (e.g. Sourcegraph has compiler-based code search) openings on-topic?

It would be helpful to have some explicit wording with examples on what kinds of openings are on-topic and off-topic.

**Comment by mgaudet** [comment]
Yeah, this is for sure something I struggled with. It's hard to document the bounds directly, as they're gut-driven, but the project would certainly benefit from me writing down some guidelines. 

To your examples: 

* Thus far I've kept Replay out: my understanding of the project is that the runtime there work is adjacent, and so I've not listed them.
* For SourceGraph, I'd be inclined to include them. 

One of the ways I filter candidates is if I find job postings that look aligned; so if I saw a SourceGraph posting that was aligned, it would help push them over the edge.
